ember-cli-build.js

```diff
@@ -46,12 +46,12 @@ module.exports = function(defaults) {
     app.import("vendor/quill/dist/quill.min.js");
     app.import("vendor/messenger/build/js/messenger.min.js");
     // app.import("node_modules/fullcalendar/dist/locale-all.js");
-    app.import("node_modules/chart.js/dist/Chart.min.js");
+    app.import("node_modules/chart.js/dist/chart.js");
```

Comment:
Why do we need to import the actual JS rather than the minified version?

Reply:
This is resolved now .. Kindly review again..